ODIN_II/parse_odin_stats.py

```diff
@@ -0,0 +1,948 @@
+        print("e.g.\t[./exec <-mc> (optional)<--flags/-f> (opt)<--concise> <output.csv> <input-1> ... <input-n>]")
+        print("options: --flags")
+        print("-con or --concise: don't print diff summary")
+        print("-------------------------------------")
+        print("COMPARE MODE: -c")
+        print("e.g.\t./parse_odin_stats.py <-c> <output.csv> <input.log> <input2.log>")
+        print("Compare two stat_gen outputs, write both metrics and diff on csv, creates flag file for results that differ significantly from (optional) threshold file, print summary of diff to stdout")
+        print("-------------------------------------")
+        print("THRESHOLD GENERATOR MODE:")
+        print("Specify +/- % of acceptable change and optionally threshold file (else it will be default filename) to generate file to be used in compareMode")
+        print("\t-Will generate same ratio for certain core stats, otherwise must customize your own threshold file with metrics you want and ratio")
+        print("\t-Format is metric_name: <ratio>\nmetric_name2: <ratio>")
+        print("-------------------------------------")
+        print("SPECIFY THRESHOLD FILE:")
+        print("\tInclude [-t <threshold_filename>] to either compare mode before input/output to include threshold file to test against")
+        print("-------------------------------------")
+        print("SIMPLE_MODE:")
+        print("Take output of stat_gen as input, parse it and place results as one line in csv/tsv")
+        print("e.g.\t[./parse_odin_stats.py <input.log> <output.csv>]\n")
+        
+    elif(code == 1):
+        print("Nothing to compare, use -mc mode")
+
+    elif(code == 2):
+        print("\tNo inputs and/or outputs specified, expecting [./exec <-m/mc> (optional)<--flags/-f> (opt)<--concise> <output> <input-1> ... <input-n>]")
+
+    elif(code == 3):
+        print("Incorrect threshold generator usage. Acceptable patterns below:")
+        print("\n1) [./exec -gen] -> yields thresholds.txt with every value as 0.15 ('-g' also accepted)")
+        print("2) [./exec -gen <ratio>] (e.g. -gen 0.25) -> specify ratio to generate thresholds.txt with custom ratio")
+        print("3) [./exec -gen <ratio> <outputfilename.txt>] -> yields specified threshold file name with specified ratio (custom name requires custom ratio)\n")
+
+    elif(code==4):
+        print("No input specified")
+    elif(code==5):
+        print("Formatting error: check log file")
+    elif(code==6):
+        print("Wrong number of arguments, expecting [./exec -c <input.log> <input2.log> (opt)<output.csv>]")
+    elif(code==7):
+        # this is our only mechanism against the possibility of overwriting the log file, important that input files are .log
+        print("\nImproper output format, output file must not begin with '-' or end in .log, use no args or -h/--help for proper format")
+
```

Comment:
rather than multiple print statement use
```
print '''
your multiline
text
goes here
'''
```

Reply:
move the help text as close to the top of this file ( maybe make an array of multiline string at the top and use the code as the index) this way it documents the usage of this python script. It will be the first thing you see when you open the file